File: algorithm/src/main/java/baekjoon/p05554/Main.java
package baekjoon.p05554;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] inputs = new int[4];

        for (int i = 0; i < inputs.length; i++) {
            inputs[i] = sc.nextInt();
        }

        for (int solve : solve(inputs)) {
            System.out.println(solve);
        }
    }

    private static int[] solve(int[] inputs) {
        int sum = 0;
        for (int n : inputs) {
            sum += n;
        }

        return new int[]{
                sum / 60,
                sum % 60
        };
    }

}

File: algorithm/src/main/java/baekjoon/p05554/Main.kt
package baekjoon.p05554

fun main(args: Array<String>) {
    val inputs = IntArray(4) { readLine()!!.toInt() }
    println(solve(inputs).joinToString("\n"))
}

private fun solve(inputs: IntArray): IntArray {
    return intArrayOf(
            inputs.sum() / 60,
            inputs.sum() % 60
    )
}